client/python/README.md

```diff
@@ -22,7 +29,7 @@ In the repo level, run `make python`
     - Package is defined by pyproject.toml
     - `poetry install` will pull dependencies and install based on pyproject.toml
 3) tox
-    - `poetry run tox -e format` will check formating/linter for your code according to default black settings and flake8
+    - `poetry run tox -e format` will check formatting/linter for your code according to default black settings and flake8
```

Comment:
This whole second will need some de-poetry-ing love. I think you can replace `poetry install` on line 30 with `pip install .`, and just remove the `poetry run` prefix from all these tox commands.

Reply:
@kindyluv doesn't look like this comment has been addressed